app/scripts/modules/core/src/managed/ManagedResourceStatusIndicator.tsx

```diff
@@ -177,30 +185,125 @@ const logClick = (label: string, resourceId: string, status: ManagedResourceStat
     label: `${resourceId},${status}`,
   });
 
-const LearnMoreLink = ({ id, status }: { id: string; status: ManagedResourceStatus }) => (
-  <p className="sp-margin-m-top sp-margin-xs-bottom">
-    <a
-      target="_blank"
-      onClick={() => logClick('Status docs link', id, status)}
-      href={`https://www.spinnaker.io/reference/managed-delivery/resource-status/#${status.toLowerCase()}`}
-    >
-      Learn more about this
-    </a>
-  </p>
+const toggleResourcePause = (
+  resourceSummary: IManagedResourceSummary,
+  application: Application,
+  hidePopover: () => void,
+) => {
+  hidePopover();
+  const { id, isPaused } = resourceSummary;
+  const toggle = () =>
+    isPaused ? ManagedWriter.resumeResourceManagement(id) : ManagedWriter.pauseResourceManagement(id);
+
+  const submitMethod = () => toggle().then(() => application.managedResources.refresh(true));
+
+  return ReactInjector.confirmationModalService.confirm({
+    header: `Really ${isPaused ? 'resume' : 'pause'} resource management?`,
+    body: getToggleBodyText(resourceSummary),
+    account: resourceSummary.locations.account,
+    buttonText: `${isPaused ? 'Resume' : 'Pause'} management`,
+    submitMethod,
+  });
+};
+
+const getToggleBodyText = (resourceSummary: IManagedResourceSummary) => {
+  const { isPaused, locations, status } = resourceSummary;
+  const regions = locations.regions.map(r => r.name).sort();
+  let body = '';
+  if (!isPaused) {
+    body += `
+        <p>
+          While a resource is paused, Spinnaker will not attempt to reconcile any differences between the declared
```

Comment:
I've been using the wording `take action to resolve drift from the declarative configuration`, rather than using the terms "reconcile" and "declared" — do you think that's good / bad? I don't have a strong preference either way as long as it's consistent between views.

Reply:
Agreed, consistency is better - if we want to reword it later, we can. I'll make this change and the others suggested.